Javascript2-week2/functionsMovieApp.js

```diff
@@ -0,0 +1,337 @@
+import { movies } from "./moviesList.js";
+import { allComments, addCommentToStorage } from './commentsFunctions.js';
+
+/* FILTERS */ // (UPDATED JS2 WK2) - Added filters section
+
+// Filters movies by genre
+function filterMoviesByGenre(genre) {
+    const filteredMovies = movies.filter(movie => movie.genres.includes(genre));
+    displayMoviesGrid(filteredMovies);
+}; 
+    // Genre filters
+function setupGenreFilters() {
```

Comment:
You are using` [...new Set()]` and sorting separately, which is unnecessary. You can use` reduce()` to collect genres in a Set and then sort.